web-admin/src/components/projects/dashboards.ts

```diff
@@ -72,3 +78,66 @@ export function getDashboardListItemsFromFilesAndCatalogEntries(
 
   return dashboardListings;
 }
+
+export function useDashboardListItems(
+  instanceId: string,
+  project: CreateQueryResult<V1GetProjectResponse>
+): Readable<{
+  items: DashboardListItem[];
+  success: boolean;
+}> {
+  let isProfiling = false;
+  if (project) {
+    const status = get(project)?.data?.prodDeployment?.status;
+    if (
+      status === "DEPLOYMENT_STATUS_PENDING" ||
+      status === "DEPLOYMENT_STATUS_RECONCILING"
```

Comment:
Again, should be e.g. `V1DeploymentStatus.DEPLOYMENT_STATUS_RECONCILING` rather than hardcoded strings.